addon/components/hello-sign.js

```diff
@@ -136,22 +134,20 @@ export default Component.extend({
       options.container = this.get('containerElement');
     }
 
-    HelloSign.open($.extend(options, {
-      messageListener: function(eventData) {
-        switch(eventData.event) {
-          case HelloSign.EVENT_SIGNED:
-            self.sendAction('onEventSigned', eventData);
-            break;
-          case HelloSign.EVENT_CANCELED:
-            self.sendAction('onEventCanceled');
-            break;
-          case HelloSign.EVENT_ERROR:
-            self.sendAction('onEventError', eventData);
-            break;
-          default:
-            self.sendAction('onEventInvalid');
-        }
+    options.messageListener = (eventData) => {
+      switch(eventData.event) {
+        case HelloSign.EVENT_SIGNED:
+          this.sendAction('onEventSigned', eventData); // eslint-disable-line
```

Comment:
Will this be a breaking change to move these to closure actions?

Reply:
No, it won't. I just made the update to remove the `sendAction`s.